runtime.md

```diff
@@ -529,13 +532,36 @@ procedure P()
 9.6 Return statements
 ---------------------
 
-```k
+```verification
+    rule <k> return ; ~> _
+          => assert { :code "BP5003" } { :source "???", 0 } { :procedure CurrentProc, CurrentImpl }
+                     substitute( Ensures
+                               , IdsTypeWhereListToIdList(PArgs) ++IdList IdsTypeWhereListToIdList(PRets)
+                               , IdsTypeWhereListToExprList(IArgs) ++ExprList IdsTypeWhereListToExprList(IRets)
+                               ) ;
+         </k>
+         <currentImpl> CurrentImpl </currentImpl>
+         <procName> CurrentProc </procName>
+         <iargs> IArgs </iargs>
+         <irets> IRets </irets>
+         <implId> CurrentImpl </implId>
+         <posts> Ensures </posts>
+         <args> PArgs </args>
+         <rets> PRets </rets>
+```
+
+TODO: this first rule is identical to the one above except for the ~> #return stuff at the end.
+Can we figure out a way to not duplicate code?
+
+```operational
```

Comment:
this first rule is identical to the one above in the verification semantics, except for the ~> #return stuff at the end.
Can we figure out a way to not duplicate code?

Reply:
We could wrap that one line that's different in a operational-only code block, but that would split the rule over three code blocks and would be pretty ugly.